src/tabs/styled-components.js

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index.js';
+import type {SharedStylePropsT} from './types.js';
+import {ORIENTATION} from './constants.js';
+
+/**
+ * Main component container element
+ */
+export const Root = styled('div', (props: SharedStylePropsT) => {
+  const {$orientation} = props;
+  return {
+    display: 'flex',
+    flexDirection: $orientation === ORIENTATION.vertical ? 'row' : 'column',
+  };
+});
+
+export const Tab = styled('div', (props: SharedStylePropsT) => {
+  const {
+    $disabled,
+    $active,
+    $orientation,
+    $theme: {colors, sizing, typography},
+  } = props;
+  let style = {
+    ...typography.font300,
+    boxSizing: 'border-box',
+    color: $active ? colors.black : colors.mono800,
+    cursor: $disabled ? 'not-allowed' : 'pointer',
+    height: sizing.scale1200,
```

Comment:
Do we actually want the height to be fixed? In the case of a very long title it seems not to work well.